### PR TITLE
Create Taichi Lang Weekly 20220420

### PR DESCRIPTION
**Pull request by lucywsq**

### CODE REVIEW

**Comment by lucywsq** [comment]
PTAL @gingerkidney @ailzhang @Vissidarte-Herman Thanks a lot!

File: weekly/2022-04-21-Taichi-Lang-Weekly.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+### **Call for participation**
+
+In this section, we highlight tasks from Taichi-Lang projects.
+
+Tags with [good first issue](https://github.com/taichi-dev/taichi/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) and [welcome contribution](https://github.com/taichi-dev/taichi/issues?q=is%3Aopen+is%3Aissue+label%3A%22welcome+contribution%22) are suitable for beginners. 
+
+### **Others**
+
+- @[Trinkle23897](https://github.com/Trinkle23897) launched a toy project [**called Fast-Poisson-Image-Editing**](https://github.com/Trinkle23897/Fast-Poisson-Image-Editing) with Taichi extension.
```
Comment (Vissidarte-Herman) [inline weekly/2022-04-21-Taichi-Lang-Weekly.md:73]:
```suggestion
- @[Trinkle23897](https://github.com/Trinkle23897) launched a fun project [**called Fast-Poisson-Image-Editing**](https://github.com/Trinkle23897/Fast-Poisson-Image-Editing) using Taichi as its extension. 
```

**Review by Vissidarte-Herman — CHANGES_REQUESTED** [review]

**Comment by Vissidarte-Herman** [comment]
@lucywsq Please add ailing and yanqing as reviewer.

File: weekly/2022-04-21-Taichi-Lang-Weekly.md (hunk 1 above)
Comment (lucywsq) [reply weekly/2022-04-21-Taichi-Lang-Weekly.md]: Done👌Thanks a lot.

**Review by gingerkidney — APPROVED** [review]

File: weekly/2022-04-21-Taichi-Lang-Weekly.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+### **Call for participation**
+
+In this section, we highlight tasks from Taichi-Lang projects.
+
+Tags with [good first issue](https://github.com/taichi-dev/taichi/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) and [welcome contribution](https://github.com/taichi-dev/taichi/issues?q=is%3Aopen+is%3Aissue+label%3A%22welcome+contribution%22) are suitable for beginners. 
+
+### **Others**
+
+- @[Trinkle23897](https://github.com/Trinkle23897) launched a fun project [**called Fast-Poisson-Image-Editing**](https://github.com/Trinkle23897/Fast-Poisson-Image-Editing) using Taichi as its extension.
```
Comment (gingerkidney) [inline weekly/2022-04-21-Taichi-Lang-Weekly.md:73]:
```suggestion
- @[Trinkle23897](https://github.com/Trinkle23897) launched a fun project called [**Fast-Poisson-Image-Editing**](https://github.com/Trinkle23897/Fast-Poisson-Image-Editing) using Taichi as its extension. 
```

File: weekly/2022-04-21-Taichi-Lang-Weekly.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+In this section, we highlight tasks from Taichi-Lang projects.
+
+Tags with [good first issue](https://github.com/taichi-dev/taichi/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) and [welcome contribution](https://github.com/taichi-dev/taichi/issues?q=is%3Aopen+is%3Aissue+label%3A%22welcome+contribution%22) are suitable for beginners. 
+
+### **Others**
+
+- @[Trinkle23897](https://github.com/Trinkle23897) launched a fun project called [**Fast-Poisson-Image-Editing**](https://github.com/Trinkle23897/Fast-Poisson-Image-Editing) using Taichi as its extension. 
+
+### **Blogs & upcoming events**
+* 2022/04/21 | Online | Meetup
```
Comment (gingerkidney) [inline weekly/2022-04-21-Taichi-Lang-Weekly.md:76]: should add voxel challenge

Comment (gingerkidney) [reply weekly/2022-04-21-Taichi-Lang-Weekly.md:76]: I suggest adding the voxel challenge at the top of the weekly

**Pull request merged**